### ISSUE TITLE
Solution?

### ISSUE THREAD

**Issue by zkendall** [danielcovill/typescript-issue#1]
Did you ever find a solution?

**Comment by danielcovill** [comment]
Yes, the issue was a result of trying to compile typescript using improper command line arguments. Someone on reddit helped me find the answer: https://www.reddit.com/r/typescript/comments/6y9khe/trouble_compiling_promises/

The rough summary is as follows:

> tsconfig.json is only used if there are no supplied parameters. You would have to do
> `tsc --target es6 settings.ts`

> Read https://www.typescriptlang.org/docs/handbook/tsconfig-json.html
> > When input files are specified on the command line, tsconfig.json files are ignored.

Hope that helps!
